electrum/transaction.py

```diff
@@ -203,6 +204,8 @@ def __init__(self, *,
         self.nsequence = nsequence
         self.witness = witness
 
+        self.multisig_script_generator = None
```

Comment:
Don't you think that making a separate transaction type, which would inherit from standard transaction, would be a much cleaner solution?

Reply:
I'm afraid that it will imply more changes in code. I case on inheritance we need to rewrite e.q `input_script` method which will be almost identical with base method. It will produce more redundant code. Generally, specific transaction class is better solution than current implementation, however I would left it as a code refactor part for the future.